pkgs/games/minetest/default.nix

```diff
@@ -118,18 +118,14 @@ let
       description = "Infinite-world block sandbox game";
       license = licenses.lgpl21Plus;
       platforms = platforms.linux ++ platforms.darwin;
-      maintainers = with maintainers; [ pyrolagus fpletz ];
-      # never built on Hydra
-      # https://hydra.nixos.org/job/nixpkgs/trunk/minetestclient_4.x86_64-darwin
-      # https://hydra.nixos.org/job/nixpkgs/trunk/minetestserver_4.x86_64-darwin
-      broken = (lib.versionOlder version "5.0.0") && stdenv.isDarwin;
```

Comment:
```
-- fixup_bundle
--   app='/nix/store/dmivkm16n092zdx9zaplwxvwm56hj9yx-minetest-5.6.1/minetest.app'
--   libs=''
--   dirs='/nix/store/dmivkm16n092zdx9zaplwxvwm56hj9yx-minetest-5.6.1/minetest.app/Contents/MacOS'
--   ignoreItems=''
-- fixup_bundle: preparing...
-- 
warning: cannot resolve item '/usr/lib/libSystem.B.dylib'
  possible problems:
    need more directories?
    need to use InstallRequiredSystemLibraries?
    run in install tree instead of build tree?
-- 
warning: cannot resolve item '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation'
  possible problems:
    need more directories?
    need to use InstallRequiredSystemLibraries?
    run in install tree instead of build tree?
-- warning: gp_resolved_file_type non-absolute file '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation' returning type 'other' -- possibly incorrect
-- 
warning: cannot resolve item '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation'
  possible problems:
    need more directories?
    need to use InstallRequiredSystemLibraries?
    run in install tree instead of build tree?
-- 
warning: cannot resolve item '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation'
  possible problems:
    need more directories?
    need to use InstallRequiredSystemLibraries?
    run in install tree instead of build tree?
-- warning: gp_resolved_file_type non-absolute file '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation' returning type 'other' -- possibly incorrect
-- 
warning: cannot resolve item '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation'
  possible problems:
    need more directories?
    need to use InstallRequiredSystemLibraries?
    run in install tree instead of build tree?
-- warning: gp_resolved_file_type non-absolute file '@rpath/CoreFoundation.framework/Versions/A/CoreFoundation' returning type 'other' -- possibly incorrect
-- 
warning: cannot resolve item '/usr/lib/libSystem.B.dylib'
  possible problems:
    need more directories?
    need to use InstallRequiredSystemLibraries?
    run in install tree instead of build tree?
CMake Error at /nix/store/irh7xcyxc6xn5wx0d075dzkvz3w8in86-cmake-3.24.1/share/cmake-3.24/Modules/BundleUtilities.cmake:458 (message):
  otool -l failed: 1
  error:
  /nix/store/bv67fzhlq7pbalq0wmzifg5ph159aqph-cctools-binutils-darwin-949.0.1/bin/otool:
  can't open file: /usr/lib/libSystem.B.dylib (No such file or directory)
Call Stack (most recent call first):
  /nix/store/irh7xcyxc6xn5wx0d075dzkvz3w8in86-cmake-3.24.1/share/cmake-3.24/Modules/BundleUtilities.cmake:527 (get_item_rpaths)
  /nix/store/irh7xcyxc6xn5wx0d075dzkvz3w8in86-cmake-3.24.1/share/cmake-3.24/Modules/BundleUtilities.cmake:649 (set_bundle_key_values)
  /nix/store/irh7xcyxc6xn5wx0d075dzkvz3w8in86-cmake-3.24.1/share/cmake-3.24/Modules/BundleUtilities.cmake:934 (get_bundle_keys)
  src/cmake_install.cmake:54 (fixup_bundle)
  cmake_install.cmake:98 (include)
FAILED: CMakeFiles/install.util 
cd /tmp/nix-build-minetest-5.6.1.drv-0/source/build && /nix/store/irh7xcyxc6xn5wx0d075dzkvz3w8in86-cmake-3.24.1/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
error: builder for '/nix/store/rqnmcymxklffig7zdbkd6447ircfqfa5-minetest-5.6.1.drv' failed with exit code 1;
```